api/src/wfl/log.clj

```diff
@@ -51,98 +103,57 @@
     (-write [_ _])))
 
 (def stdout-logger
-  "A logger to write to standard output"
+  "A logger to write to standard output."
   (reify Logger
     (-write [_logger edn]
-      (let [json-write-str
-            #(json/write-str
-              %
-              :escape-slash false
-              :key-fn       key-fn)
-            to-log
-            (try (json-write-str edn)
-                 (catch Throwable t
-                   (json-write-str {:tried-to-log (str edn)
-                                    :cause        (str t)})))]
-        (println to-log)))))
+      (println (try (json/write-str edn
+                                    :escape-slash false
+                                    :key-fn       key-fn)
+                    (catch Throwable t
+                      (pr-str (str {:tried-to-log edn
```

Comment:
Is `(pr-str (str … ))` intentional?

Reply:
Yes, but I can try again without the double-stringing.

Reply:
A single `(str …)` generates a string of course, but the resulting string cannot be parsed as JSON --
not by `clojure.data.json`, at least.
`str` just quotes the thing while escaping quotes and newlines, so keys are unquoted EDN keywords.
A single `(pr-str …)` has the same result because it prints the EDN as an EDN string,
which also cannot be parsed as JSON.
`(str (str …))` fails similarly, because once a string, always a string.
`(pr-str (str …))` results in a JSON string that `clojure.data.json` can parse, because it first quotes
the EDN and then encodes that as an EDN string literal, which can also parse as a JSON string (containing EDN).
You could swap them around, but then everything winds up double-escaped, which doesn't help either.
But `(pr-str … (str …))` leaves us with an EDN string that can be parsed as a JSON string, and then again
correctly parsed as EDN so we can decode it later.
Try it.